protos/google/cloud/scheduler/v1/cloudscheduler.proto

```diff
@@ -0,0 +1,237 @@
+// Copyright 2019 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package google.cloud.scheduler.v1;
+
+import "google/api/annotations.proto";
+import "google/cloud/scheduler/v1/job.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+
+option go_package = "google.golang.org/genproto/googleapis/cloud/scheduler/v1;scheduler";
+option java_multiple_files = true;
+option java_outer_classname = "SchedulerProto";
+option java_package = "com.google.cloud.scheduler.v1";
+option objc_class_prefix = "SCHEDULER";
+
+
+// The Cloud Scheduler API allows external entities to reliably
+// schedule asynchronous jobs.
+service CloudScheduler {
+  // Lists jobs.
+  rpc ListJobs(ListJobsRequest) returns (ListJobsResponse) {
+    option (google.api.http) = {
+      get: "/v1/{parent=projects/*/locations/*}/jobs"
+    };
+  }
+
+  // Gets a job.
+  rpc GetJob(GetJobRequest) returns (Job) {
+    option (google.api.http) = {
+      get: "/v1/{name=projects/*/locations/*/jobs/*}"
+    };
+  }
+
+  // Creates a job.
+  rpc CreateJob(CreateJobRequest) returns (Job) {
+    option (google.api.http) = {
+      post: "/v1/{parent=projects/*/locations/*}/jobs"
+      body: "job"
+    };
+  }
+
+  // Updates a job.
+  //
+  // If successful, the updated [Job][google.cloud.scheduler.v1.Job] is returned. If the job does
+  // not exist, `NOT_FOUND` is returned.
+  //
+  // If UpdateJob does not successfully return, it is possible for the
+  // job to be in an [Job.State.UPDATE_FAILED][google.cloud.scheduler.v1.Job.State.UPDATE_FAILED] state. A job in this state may
+  // not be executed. If this happens, retry the UpdateJob request
+  // until a successful response is received.
```

Comment:
~I wonder if it might be worth adding a comment that links to the corresponding online docs for each method? (even though it is pretty clear from the path in `patch`, `delete`, etc.~

never mind, I see that these endpoints are well documented in cloud_scheduler_client.js.